skglm/datafits/single_task.py

```diff
@@ -126,6 +126,11 @@ def get_spec(self):
     def params_to_dict(self):
         return dict()
 
+    def raw_hessian(self, y, Xw):
+        """Compute hessian of datafit w.r.t ``Xw``."""
+        exp_yXw = np.exp(-y * Xw)
+        return exp_yXw / (1 + exp_yXw) ** 2 / len(y)
```

Comment:
numba does not have https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.expit.html#scipy-special-expit ?

Reply:
Yes indeed. But unfortunately, numba is not compatible with ``scipy`` and thereby throws an error at JIT compilation time.